src/interface.c

```diff
@@ -122,6 +122,8 @@ int hashconfig_init (hashcat_ctx_t *hashcat_ctx)
 
   if (rc_load == false) return -1;
 
+  module_ctx->module_usage_notice = MODULE_DEFAULT; // set all module to have usage_notice by empty default; such that this property doesn't have to be declared explicitly by all modules (such that we don't break private plugins without this options)
```

Comment:
I kind of like the idea that we no longer need to update all existing plugins because of a newly added callback, however it's the first time we would do this. Did you check if this works on windows, too?